server/mscalendar/views/calendar.go

```diff
@@ -93,17 +94,26 @@ func renderEvent(event *remote.Event, asRow bool, timeZone string) (string, erro
 
 	format := "(%s - %s) [%s](%s)"
 	if asRow {
-		format = "| %s - %s | [%s](%s) |"
+		format = "| %s - %s | [%s](%s) | %s |"
 	}
 
 	link, err := url.QueryUnescape(event.Weblink)
 	if err != nil {
 		return "", err
 	}
 
+	var other string
+	if event.Location != nil && isKnownMeetingURL(event.Location.DisplayName) {
+		other = "[Join meeting](" + event.Location.DisplayName + ")"
+	}
+
 	subject := EnsureSubject(event.Subject)
 
-	return fmt.Sprintf(format, start, end, subject, link), nil
+	return fmt.Sprintf(format, start, end, subject, link, other), nil
+}
+
+func isKnownMeetingURL(location string) bool {
+	return strings.Contains(location, "zoom.us/j/") || strings.Contains(location, "discord.gg") || strings.Contains(location, "meet.google.com")
```

Comment:
Neat :+1:  Something comes to mind, that a customer may have other call-related domains, including Mattermost Calls. Maybe have this configurable in some way?

Reply:
Yeah, I want to integrate calls too, but need to send the SITE_URL over to check the URL to do it properly, so it needs a bit more work. Also, Google sends `conferenceData` for some events instead of location (I'm assuming they have a specific integration), so it wont work every time.